source/features/toggle-files-button.tsx

```diff
@@ -25,20 +27,24 @@ function addButton(): void {
 			type="button"
 			className="btn-octicon rgh-toggle-files"
 			aria-label="Toggle files section"
-			aria-expanded="true"
 		>
 			<FoldIcon/>
 			<UnfoldIcon/>
 		</button>
 	);
 }
 
-function init(): void {
-	const repoContent = select('.repository-content')!;
+async function init(): Promise<void> {
+	const cacheKey = 'files-list-toggled-off';
+	const repoContent = (await elementReady('.repository-content'))!;
 	observeElement(repoContent, addButton);
-	delegate(document, '.rgh-toggle-files', 'click', ({delegateTarget}) => {
-		delegateTarget.setAttribute('aria-expanded', String(!repoContent.classList.toggle('rgh-files-hidden')));
+
+	delegate(document, '.rgh-toggle-files', 'click', async () => {
+		await cache.set(cacheKey, repoContent.classList.toggle('rgh-files-hidden'));
 	});
+	if (await cache.get<boolean>(cacheKey)) {
+		select('.rgh-toggle-files')!.click();
```

Comment:
We can return to the plain class toggle now that it's just one instruction anyway
```suggestion

	if (await cache.get<boolean>(cacheKey)) {
		repoContent.classList.toggle('rgh-files-hidden');
```